src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -115,25 +115,18 @@ void
 FinishRemoteTransactionBegin(struct MultiConnection *connection)
 {
 	RemoteTransaction *transaction = &connection->remoteTransaction;
-	PGresult *result = NULL;
-	const bool raiseErrors = true;
+	bool clearSuccessful = true;
 
 	Assert(transaction->transactionState == REMOTE_TRANS_STARTING);
 
-	result = GetRemoteCommandResult(connection, raiseErrors);
-	if (!IsResponseOK(result))
-	{
-		ReportResultError(connection, result, WARNING);
-		MarkRemoteTransactionFailed(connection, raiseErrors);
-	}
-	else
+	clearSuccessful = ClearRemainingResults(connection);
```

Comment:
It seems like it should actually be called `ClearResults`, since there isn't a remaining here. It's all results.

Reply:
done